docs/resources/shared_environment_variable_project_link.md

```diff
@@ -0,0 +1,44 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "vercel_shared_environment_variable_project_link Resource - terraform-provider-vercel"
+subcategory: ""
+description: |-
+  Links a project to a Shared Environment Variable.
+  ~> This resource is intended to be used alongside a vercel_shared_environment_variable Data Source, not the Resource. The Resource also defines which projects to link to the shared environment variable, and using both vercel_shared_environment_variable and vercel_shared_environment_variable_project_link together results in undefined behavior.
+---
+
+# vercel_shared_environment_variable_project_link (Resource)
+
+Links a project to a Shared Environment Variable.
```

Comment:
Or does it link a shared environment variable to a project? :trollface:

Reply:
i've heard it both ways